### PR TITLE
chore: add pulse effect + update headsup text

### LINKED ISSUE #701: Donation processing modal polish
https://www.loom.com/share/a7cd1e4279ff4ca28c422e8e6d654ea6
- add pulsing indicator
- revise language around multiple transactions

**User Story:**
As a
I want to
So that

**Acceptance Criteria**
GIVEN
WHEN
THEN

**Product & Design Links:**

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- add pulse effect
- update headsup text

##### Refers/Fixes

fixes #701

### CODE REVIEW

**Review by bhargavaparoksham — APPROVED** [review]

File: packages/grant-explorer/tailwind.config.js [hunk 1]
```diff
@@ -58,6 +61,14 @@ module.exports = {
             color: "#FFF",
           },
         },
+        "pulse-scale": {
```
Comment (thelostone-mc) [inline packages/grant-explorer/tailwind.config.js:64]: Oh nice catch 🙌🏼

**Review by hmrtn — APPROVED** [review]

**Review by melissa-neira — DISMISSED** [review]
The icon in the heads up pop-up is a bit squished - it should be a 40x40 circle
Screen Shot 2022-12-07 at 12 25 16

**Comment by thelostone-mc** [comment]
> The UI on this is really jumpy — the processing modal flashes for half a second and then goes straight to the thank you page. Can we smooth that out?

Could we track it on a new ticket, cause this looks an existing issue in the UI when you try to contribute to a round which ahs already ended

**Comment by bhargavaparoksham** [comment]
@nfrgosselin This should get fixed as part of https://github.com/gitcoinco/grants-round/issues/751 in pr https://github.com/gitcoinco/grants-round/pull/739

**Comment by bhargavaparoksham** [comment]
@nfrgosselin This will be fixed as part of #751 in pr #636

**Comment by thelostone-mc** [comment]
@bhargavaparoksham killing it 🏅

**Review by nfrgosselin — APPROVED** [review]

**Comment by nfrgosselin** [comment]
@thelostone-mc @bhargavaparoksham ok great! just approved from my end

**Pull request merged**